tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -0,0 +1,77 @@
+@api @skipOnOcV10
+Feature: get users
+  As an admin
+  I want to be able to retrieve user information
+  So that I can see the information
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario: admin user gets the information of a user
+    When user "Alice" tries to get information of user "Brian" using Graph API
```

Comment:
```suggestion
    When user "Alice" gets information of user "Brian" using Graph API
```
As per the scenario, this will be much more logical. Note: You don't need to implement it separately. We can add it to the same implementation.